Chrono/Chrono/Splash Screen/SplashScreen.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  SplashView.swift
+//  Chrono
+//
+//  Created by Sagar on 2021-06-21.
+//
+
+import SwiftUI
+
+
+struct SplashScreen: View {
+    
+    @State var isActive:Bool = false
+
+    var body: some View {
+        VStack {
+            if self.isActive {
+                TabBar()
+            } else {
+                SplashView()
+            }
+        }
+        .onAppear {
+            DispatchQueue.main.asyncAfter(deadline: .now() + 1.0) {
+                withAnimation {
+                    self.isActive = true
+                }
+            }
+        }
+    }
+    
+}
+
+struct SplashView: View {
+    var body: some View {
+        ZStack {
+            LinearGradient(gradient: Gradient(colors: [Color(hex: "7E51FF"), Color(hex: "0E1C9F")]), startPoint: .top, endPoint: .bottom)
```

Comment:
What about dark mode? On the Android app Dark mode has a different splash screen design (it's on Figma I believe)

Reply:
Totally didn't see the dark mode version. Just added it 

![image](https://user-images.githubusercontent.com/15317925/123021758-ab7a1700-d3a2-11eb-9cb0-5bdbda5d4cd6.png)